src/control/states/DraggingState.ts

```diff
@@ -45,20 +46,19 @@ class DraggingState extends State {
       axesEvent
     });
 
-    if (flicking.getRenderer().getPanelCount() <= 0) {
+    if (flicking.renderer.getPanelCount() <= 0) {
       // There're no panels
       transitTo(STATE_TYPE.IDLE);
       return;
     }
 
-    const control = flicking.getControl();
-    const controller = control.getController()!;
-
     transitTo(STATE_TYPE.ANIMATING);
 
-    controller.setReleaseContext(axesEvent);
-    control.moveToPosition(axesEvent.destPos.flick, Math.max(axesEvent.duration, flicking.getDuration()));
-    controller.setReleaseContext(null);
+    const control = flicking.control;
+    const position = axesEvent.destPos.flick;
+    const duration = Math.max(axesEvent.duration, flicking.duration);
+
+    void control.moveToPosition(position, duration, axesEvent);
```

Comment:
Why need to `void`?

Reply:
Because of the new eslint lint rule that requires `await` or `void` to all promise-based methods.
We can discuss whether to use it, but I think it's neat as it's more revealing that it's an asynchronous method.